ocaml-lsp-server/src/custom_requests/req_documentation.mli

```diff
@@ -0,0 +1,38 @@
+val meth : string
+
+val capability : string * [> `Bool of bool ]
+
+module GetDocClientCapabilities : sig
+  type t = { contentFormat : Import.MarkupKind.t list option }
+
+  val yojson_of_t :
+    t -> [> `Assoc of (string * [> `List of Yojson.Safe.t list ]) list ]
+
+  val t_of_yojson : Yojson.Safe.t -> t
+end
+
+module GetDocParams : sig
+  type t =
+    { text_document : Import.TextDocumentIdentifier.t
```

Comment:
`open Import` should be done in the beginning of your module and not qualified explicitly